TMDS_3/include/CGAL/facets_in_complex_3_to_triangle_mesh.h

```diff
@@ -0,0 +1,245 @@
+// Copyright (c) 2009-2017 GeometryFactory (France).
+// All rights reserved.
+//
+// This file is part of CGAL (www.cgal.org).
+//
+// $URL$
+// $Id$
+// SPDX-License-Identifier: GPL-3.0-or-later OR LicenseRef-Commercial
+//
+//
+// Author(s)     : Maxime Gimeno,
+//                 Mael Rouxel-Labbé
+
+#ifndef CGAL_FACETS_IN_COMPLEX_3_TO_TRIANGLE_MESH_H
+#define CGAL_FACETS_IN_COMPLEX_3_TO_TRIANGLE_MESH_H
+
+#include <CGAL/license/TMDS_3.h>
+
+#include <CGAL/array.h>
+#include <CGAL/boost/graph/Euler_operations.h>
+#include <CGAL/Polygon_mesh_processing/orient_polygon_soup.h>
+#include <CGAL/Polygon_mesh_processing/polygon_soup_to_polygon_mesh.h>
+#include <CGAL/Named_function_parameters.h>
+#include <CGAL/boost/graph/named_params_helper.h>
+#include <CGAL/Time_stamper.h>
+#include <CGAL/property_map.h>
+
+#include <boost/unordered_map.hpp>
+
+#include <algorithm>
+#include <cstddef>
+#include <iterator>
+#include <vector>
+
+namespace CGAL {
+
+namespace TMDS_3 {
+
+namespace internal {
+
+template <class Polygon>
+void resize(Polygon& p, std::size_t size)
```

Comment:
There also exists https://github.com/CGAL/cgal/blob/master/STL_Extension/include/CGAL/Container_helper.h#L32 (which should be updated to newer C++)